cmake/macros/macro_deal_ii_pickup_tests.cmake

```diff
@@ -81,6 +81,9 @@ MACRO(DEAL_II_PICKUP_TESTS)
   # Necessary external interpreters and programs:
   #
 
+  FIND_PACKAGE(CUDA)
+  SET(CUDA_NVCC_FLAGS ${CUDA_NVCC_FLAGS} -std=c++11 -arch=sm_35)
```

Comment:
is this `sm_35` specific to GPU or it's more like a minimum feature set like C++11?